src/index.js

```diff
@@ -0,0 +1,27 @@
+// import _ from 'lodash';
+import './style.css';
+
+export default class TodoList {
+  constructor() {
+    this.tasks = [];
+  }
+
+  addTask = (description) => {
+    const completed = false;
+    const id = this.tasks.length + 1;
+    const task = {
+      id,
+      description,
+      completed,
+    };
+    this.tasks.push(task);
+  }
```

Comment:
- [ ] Great job working on the function but you missed some requirements since you should make array of objects to display some tasks for example three or four tasks to display them so please as required add an arry of thre objects and make a function to display them in the next milestone you will do it dynamically 🔥✅✨💯

![image](https://user-images.githubusercontent.com/58553711/213034335-5c3d11e1-3511-43d4-be83-9c42fd0d27bc.png)


- It should be somthing like this just static to do list ✅
![image](https://user-images.githubusercontent.com/58553711/213034426-15e5ab3a-2180-481b-9b95-30024a20cc40.png)